pyfstat/mcmc_based_searches.py

```diff
@@ -2084,8 +2004,16 @@ def _p_val_twoFhat(self, twoFhat, ntrials, twoFhatmax=500, Npoints=1000):
         pdf = self._pdf_twoFhat(twoFhats, self.nglitch, ntrials)
         return np.trapz(pdf, twoFhats)
 
-    def get_p_value(self, delta_F0, time_trials=0):
-        """ Get's the p-value for the maximum twoFhat value """
+    def get_p_value(self, delta_F0=0, time_trials=0):
+        """Get's the p-value for the maximum twoFhat value assuming Gaussian noise
```

Comment:
```suggestion
        """Gets the p-value for the maximum twoFhat value assuming Gaussian noise
```

(It should ideally also be clearly documented what this actually means, but can be done another time.)